docs/reference/experiment-config.txt

```diff
@@ -472,21 +467,21 @@ created with the Determined CLI. It may contain the following fields:
 .. _batches-per-step:
 
 - ``batches_per_step``: Specifies the number of batches in a single
-  training step. As discussed above, Determined divides the training of a
-  single trial into a sequence of steps; each step corresponds to a
-  certain number of model updates. Therefore, this configuration
+  training workload. As discussed above, Determined divides the training of a
+  single trial into a sequence of training workloads; each workload corresponds
+  to a certain number of model updates. Therefore, this configuration
   parameter can be used to control how long a trial is trained at a
   single agent:
 
-  - Doing more work in a step allows per-step overheads (such as
+  - Doing more work in a workload allows per-workload overheads (such as
     downloading training data) to be amortized over more training
-    work. However, if the step size is too large, a single trial might
+    work. However, if the workload size is too large, a single trial might
     be trained for a long time before Determined gets an opportunity to
     suspend training of that trial and replace it with a different
     workload.
-  - The default value is 100. As a rule of thumb, the step size should
-    be set so that training a single step takes 60--180 seconds.
-  - *NOTE:* The step size is defined as a fixed number of *batches*;
+  - The default value is 100. As a rule of thumb, the training workload size
```

Comment:
So a training workload will now train a model for only 100 minibatches by default?

Reply:
Yes, by default, it will be 100 minibatches, but this is shortened in the event a validation or checkpoint is needed, or we would exceed the number of batches currently requested by the searcher.